frontend-cert/js-projects/weather-app/user-stories.md

```diff
@@ -0,0 +1,23 @@
+You should have an `img` element for displaying the weather icon.
+
+You should have a `span` element for displaying the main temperature.
+
+You should have a `span` element for displaying what the temperature feels like.
+
+You should have a `span` element for displaying the amount of humidity in air.
+
+You should have a `span` element for displaying the wind speed.
+
+You should have a `span` element for displaying the wind gust.
+
+You should have a `span` element for displaying the main weather type.
+
+You should have a `span` element for displaying the current location.
```

Comment:
It would be good to mention the `id` of the respective element so you can test them in javaScript.

For example:

- You should have a `span` element with the `id` `main-temperature` for displaying the main temperature.

- You should have a `span` element with the `id` `feels-like` for displaying how the temperature feels like.

Reply:
I thought of that but then again I thought that maybe the campers would want to name them as they see fit. But Now that you say that, yeah I should. I'll make the changes right away.

Reply:
I have implemented your suggestions, can you take another look?